src/CharacterAriaLive.js

```diff
@@ -16,50 +17,46 @@ type CharacterAriaLiveProps = {
 };
 
 class CharacterAriaLive extends React.Component<CharacterAriaLiveProps, {}> {
-    getCharacterAriaLabel() {
-        if (this.props.world === 'Space') {
-            return this.props.intl.formatMessage(
-                {id:'CharacterAriaLive.spaceShipCharacter'}
-            );
-        } else if (this.props.world === 'Jungle') {
-            return this.props.intl.formatMessage(
-                {id:'CharacterAriaLive.rabbitCharacter'}
-            );
-        } else {
-            return this.props.intl.formatMessage(
-                {id:'CharacterAriaLive.robotCharacter'}
-            );
-        }
-    }
-
     setCharacterMovingAriaLive() {
         const ariaLiveRegion = document.getElementById(this.props.ariaLiveRegionId);
-        const character = this.getCharacterAriaLabel();
 
         // $FlowFixMe: Flow doesn't know that elements have innerText.
         ariaLiveRegion.innerText=this.props.intl.formatMessage(
-            {id:'CharacterAriaLive.movementAriaLabel'},
-            { character }
+            {id:'CharacterAriaLive.movementAriaLabel'}
         );
     }
 
     updateCharacterPositionAriaLive() {
         const characterState = this.props.characterState;
-        const xPos = characterState.getColumnLabel();
-        const yPos = characterState.getRowLabel();
+        const columnLabel = characterState.getColumnLabel();
+        const rowLabel = characterState.getRowLabel();
+        const { xPos, yPos } = characterState;
         const direction = this.props.intl.formatMessage({id: `Direction.${characterState.direction}`});
         const ariaLiveRegion = document.getElementById(this.props.ariaLiveRegionId);
-        const character = this.getCharacterAriaLabel();
-        // $FlowFixMe: Flow doesn't know that elements have innerText.
-        ariaLiveRegion.innerText=this.props.intl.formatMessage(
-            {id:'CharacterAriaLive.positionAriaLabel'},
-            {
-                character,
-                xPos,
-                yPos,
-                direction
-            }
-        );
+        const backgroundInfo = getBackgroundInfo(this.props.world, xPos - 1, yPos - 1);
```

Comment:
If we went with my object based strategy, we could also avoid "off by one" thinking like this, and just let row and cell one mean the same thing for objects in the scene as they do for the character position.

Reply:
This is also exactly where you transposed the coordinates.  It should be `yPos -1` first.